components/static-pages/privacypolicy.md

```diff
@@ -53,6 +53,10 @@ We may also obtain information from other sources.  For example, if you connect
 </ul>
 </div>
   </div>
+  <div style="display:flex;flex-direction: column;margin-top:1.5em">
+    <div style="font-weight:bold;font-size:2.2rem;">Objections, Right To Be Forgotten, and Data Removal</div>
+    <div style="display:flex;flex-direction:column;">If you are unhappy with your information being processed, shared or stored in line with the content of this policy please email <a href="mailto:support@opencollective.com">support@opencollective.com</a> and we will work to remove public references to your activity on the site. Note that in order to maintain our accounting, tax and legal obligations we may be required to retain some private information about you once this process is complete. </div>
```

Comment:
This kinda breaks the text flow

```suggestion
    <div>If you are unhappy with your information being processed, shared or stored in line with the content of this policy please email <a href="mailto:support@opencollective.com">support@opencollective.com</a> and we will work to remove public references to your activity on the site. Note that in order to maintain our accounting, tax and legal obligations we may be required to retain some private information about you once this process is complete. </div>
```

## Before
![image](https://user-images.githubusercontent.com/1556356/134874443-01501493-54a2-4588-9483-b603d6533153.png)

## After
![image](https://user-images.githubusercontent.com/1556356/134874475-095db2d5-7ac1-4e0e-8d06-31e0c05c7729.png)